buildbot/configure.py

```diff
@@ -40,6 +40,7 @@ def do_configure(args):
     libdevice_dir = os.path.join(abs_src_dir, "libdevice")
     fusion_dir = os.path.join(abs_src_dir, "sycl-fusion")
     llvm_targets_to_build = args.host_target
+    llvm_experimental_targets_to_build = 'SPIRV'
```

Comment:
i wonder if @bader has an opinion on the effect on CI/other users since basically everyone except this team isn't going to use this so its wasted time/cycles for them

Reply:
I would prefer SPIR-V backend to be enabled by the option, which is disabled by default to avoid the impact on other DPC++ compiler developers. 
I'm okay with turning on this option in CI if some reasonable number of tests are passing already and we want to avoid regressions in these tests.

To be honest, I expected much more changes to make LTO framework usable for SYCL compilation flow. Specifically replacing SPIR-V translator with SPIR-V back-end should introduce some number of .

@VyacheslavLevytskyy, will it be useful for you if we enable built of SPIR-V target in our CI (e.g. gather SYCL tests pass rate on additional platforms)? NOTE: `sycl` branch is usually behind `main` branch (few days, but can be weeks).

Overall, SPIR-V backend today is relatively small, so the overhead it not significant.

Reply:
@sarnex and @bader 

Thanks so much for this discussion. One pointer is that I intend to use SPIR-V backend only for LTO with early exit condition, where we emit linked .bc file and exit early. I do not intend to use the SPIR-V backend for LLVM to SPIR-V translation at this stage.

Thanks
Sincerely

Reply:
@bader Indeed, it'd be helpful to gather info about other environments easier, even with a delay of days/weeks. This move would make sense one day, but only on condition that we introduced a couple of minor changes into the translation chain under a new option. These would include (1) diagnostics from `spirv-val` about produced SPIR-V output, and (2) verification that `llvm-spirv -r` is able to produce LLVM IR from the SPIR-V output. Until this is changed all intermediate crashes are hidden, and we see only final numbers of fail/success cases, without any tips about possible causes of fails.